types/rke_types.go

```diff
@@ -415,6 +415,12 @@ type IngressConfig struct {
 	ExtraVolumeMounts []ExtraVolumeMount `yaml:"extra_volume_mounts" json:"extraVolumeMounts,omitempty" norman:"type=array[json]"`
 	// nginx daemonset upgrade strategy
 	UpdateStrategy *DaemonSetUpdateStrategy `yaml:"update_strategy" json:"updateStrategy,omitempty"`
+	// Http port for ingress controller daemonset
+	HTTPPort int `yaml:"http_port" json:"http_port,omitempty"`
+	// Https port for ingress controller daemonset
+	HTTPSPort int `yaml:"https_port" json:"https_port,omitempty"`
+	// NetworkMode selector for ingress controller pods. Default is HostNetwork
+	NetworkMode string `yaml:"networkMode" json:"networkMode,omitempty"`
```

Comment:
yaml convention should be `network_mode`

Reply:
fixed up the tags.